src/Modules/Pages/Domain/Revision/Form/RevisionType.php

```diff
@@ -78,7 +80,7 @@ function (FormEvent $event): void {
                     'generate_slug_callback_method' => 'generateSlug',
                     'generate_slug_callback_parameters' => [
                         $revisionDataTransferObject->locale,
-                        $revisionDataTransferObject->hasEntity() ? $revisionDataTransferObject->getEntity()->getId() : null,
+                        $entity ? $entity->getId() : null,
```

Comment:
@bjorvack did you test this? Because you are trying to get an id from a boolean

Reply:
fixed cae7f5d515be39e140421b66a5740fcf72afe9cc